javascript/javascript3/week3/ShoppingCart/main.js

```diff
@@ -0,0 +1,135 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
```

Comment:
Also great work here. You don't have trouble understanding programming. 
What I wrote previously applies here as well. First of all, variable names like p are forbidden.
Secondly, avoid at all cost to declare variables at the top and then change them in various places throughout the function.
I want you to try and refactor convertToCurrency. These are rules:
- No duplication
- Calls to several different functions or maybe even a new class
- No if or else statements.
- Separate the fetch call into its own function
- 8-10 lines of code in the function at most. Preferably less